docs/rum_collection/advanced_configuration.md

```diff
@@ -0,0 +1,370 @@
+
+For instance, you can configure the predicate to use explicit type check for each view controller in your app:
+```swift
+class YourCustomPredicate: UIKitRUMViewsPredicate {
+    func rumView(for viewController: UIViewController) -> RUMView? {
+        switch viewController {
+        case is HomeViewController:     return .init(name: "Home")
+        case is DetailsViewController:  return .init(name: "Details")
+        default:                        return nil
+        }
+    }
+}
+```
+
+You can even come up with a more dynamic solution depending on your app's architecture. For example, if your view controllers use `accessibilityLabel` consistently, you can name views by the value of accessibility label:
+```swift
+class YourCustomPredicate: UIKitRUMViewsPredicate {
+    func rumView(for viewController: UIViewController) -> RUMView? {
+        if let accessibilityLabel = viewController.accessibilityLabel {
+            return .init(name: accessibilityLabel)
+        } else {
+            return nil
+        }
+    }
+}
+```
+
+**Note**: The SDK calls `rumView(for:)` many times while your app is running. It is recommended to keep its implementation fast, performant and single-threaded.
+
+### Automatically track network requests
+
+To automatically track resources (network requests) and get their timing information such as time to first byte or DNS resolution, use the `.trackURLSession()` option when configuring the SDK and set `DDURLSessionDelegate` for the `URLSession` that you want to monitor:
+```swift
+let session = URLSession(
+    configuration: .default,
+    delegate: DDURLSessionDelegate(),
+    delegateQueue: nil
+)
+```
+
+Also, you can configure first party hosts using `.trackURLSession(firstPartyHosts:)`. This will classify resources matching given domain as "first party" in RUM and will propagate tracing information to your backend (if Tracing feature is enabled).
+
+For instance, you can configure `yourdomain.com` as first party host and enable both RUM and Tracing features:
+```swift
+Datadog.initialize(
+    // ...
+    configuration: Datadog.Configuration
+        .builderUsing(/* ... */)
+        .trackUIKitRUMViews()
+        .trackURLSession(firstPartyHosts: ["yourdomain.com"])
+        .build()
+)
+
+Global.rum = RUMMonitor.initialize()
+Global.sharedTracer = Tracer.initialize()
+
+let session = URLSession(
+    configuration: .default,
+    delegate: DDURLSessionDelegate(),
+    delegateQueue: nil
+)
+```
+This will track all requests sent with the instrumented `session`. Requests matching `yourdomain.com` domain will be marked as "first party" and tracing information will be send to your backend to [connect the RUM resource with its Trace][10].
```

Comment:
```suggestion
This will track all requests sent with the instrumented `session`. Requests matching `example.com` domain will be marked as "first party" and tracing information will be send to your backend to [connect the RUM resource with its Trace][10].
```